src/AzureFunctions/Example/GraphQLExecutionResult.cs

```diff
@@ -39,10 +38,7 @@ public override async Task ExecuteResultAsync(ActionContext context)
             // Azure functions 3 disallowing async IO and newtonsoft json is not able to
             // make real async IO, we need copy to a MemoryStream.
             // After graphql has switch to System.Text.Json this can be written directly to response.Body
```

Comment:
Pay attention to this comment. I suggest to switch to STJ and remove comment.

Reply:
I do not use Azure Functions and do not have time to spend learning about them, testing them, or debugging them at this time.  Feel free to post a PR to make any changes you wish in this regard.

Reply:
I mean that comments explicitly point that your changes will break the code. Ping @StefanSchoof for advice.

Reply:
I see but the comments are conflicting, and more likely the first line should have read "disallowing sync IO" and then the whole paragraph would make sense.

Reply:
As it reads now, the comment says it disallows async IO (which is unlikely), then then says that newtonsoft cannot write async IO (which although true would not matter if async IO is disallowed), and implies that to use newtonsoft you'd need to copy to a memorystream (which would be false if async IO was disallowed).  and then the comment continues and says that with STJ, the response can be written directly to response.body (as I have it, but which would be false if async IO was disallowed)

Reply:
Ah, I see. OK to merge as is and then change code/comments after notes from @StefanSchoof .